lab-padget/app/index.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+require('./scss/reset.scss');
+require('./scss/base.scss');
+require('./scss/main.scss');
+
+const angular = require('angular');
+const cowsay = require('cowsay-browser');
+const cowsayApp = angular.module('cowsayApp', []);
+
+cowsayApp.controller('CowsayController', ['$log', CowsayController]);
+
+function CowsayController($log) {
+  $log.debug('#CowsayController');
+
+  $log.log('check this out', this);
+
```

Comment:
it's considered best practice to wrap the innards of your controller in an $onInit function so all the methods and properties are compiled and ready to be used